.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: ruby
+rvm:
+  - 1.9.3
```

Comment:
Since this is what we use internally, maybe it should be tested on multiple versions, including 1.9?